eval/src/vespa/eval/eval/value_cache/constant_tensor_loader.cpp

```diff
@@ -76,23 +79,33 @@ ConstantTensorLoader::create(const vespalib::string &path, const vespalib::strin
         LOG(warning, "invalid type specification: %s", type.c_str());
         return std::make_unique<SimpleConstantValue>(_engine.from_spec(TensorSpec("double")));
     }
-    Slime slime;
-    decode_json(path, slime);
-    std::set<vespalib::string> indexed;
-    for (const auto &dimension: value_type.dimensions()) {
-        if (dimension.is_indexed()) {
-            indexed.insert(dimension.name);
-        }
+    if (ends_with(path, ".tbf")) {
+        vespalib::File file(path);
+        file.open(File::READONLY);
+    	std::vector<char> content(file.stat()._size);
+	file.read(&content[0], content.size(), 0);
```

Comment:
is complete file read guaranteed also for docker with ant file system mounting? (mac)
(I do not know)

Reply:
Changed to MappedFileInput